deploy/package.py

```diff
@@ -5,10 +5,14 @@
 import re
 import argparse
 import shutil
-
+import json
+from pathlib import Path
 
 # Collect args
 parser = argparse.ArgumentParser(description='Arguments required to run packaging function')
+parser.add_argument('--modes',
```

Comment:
If we want to setup infra with batch-account and no-postgres, what parameter values we are setting for this?
And, do we have logic in package.sh?